build.sbt

```diff
@@ -35,6 +35,7 @@ scalacOptions in Compile ++= Seq("-feature", "-deprecation")
 addSbtPlugin("com.dwijnand" % "sbt-dynver" % "1.2.0")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.1")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.0")
```

Comment:
I really don't think we should go this far... typical cake projects are **not** published on sonatype and this just opens up the possibility of accidental error.